### PR TITLE
only show top 5 comments initially

### PR DESCRIPTION
**Pull request by jon-ec**

### CODE REVIEW

**Comment by jon-ec** [comment]
@panosvoudouris @leanne1

File: src/components/comments/index.jsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { ListGroup } from 'react-bootstrap';
 import Comment from '../comment';
 import SubmitComment from '../submitComment';
 import { getTimestamp } from '../../utils';
+import { Button } from 'react-bootstrap';
```
Comment (leanne1) [inline src/components/comments/index.jsx]: you can import this from same line as ListGroup import

File: src/components/comments/index.jsx [hunk 2]
```diff
@@ -14,6 +15,8 @@ const Comments = ({
 	onUserCommentUpdate,
 	onUserCommentEdit,
 	onUserCommentDelete,
+	onShowAllComments,
+	showingAllComments
```
Comment (leanne1) [inline src/components/comments/index.jsx]: `isShowingAllComments` maybe?

File: src/components/comments/index.jsx [hunk 3]
```diff
@@ -53,10 +56,14 @@ const Comments = ({
 			</div>
 		)
 	});
+	const showAllCommentsLink = !showingAllComments ? <Button bsStyle="link" block style={{ marginBottom: 20 }}onClick={onShowAllComments}>Show all comments</Button> : null;
```
Comment (leanne1) [inline src/components/comments/index.jsx]: I'd maybe break this over a few lines, just a suggestion:

```js
const showAllCommentsLink = !showingAllComments ? 
    <Button bsStyle="link" block style={{ marginBottom: 20 } }onClick={onShowAllComments}>
        Show all comments
    </Button> :
    null;
```

File: src/components/panel/index.jsx [hunk 4]
```diff
@@ -81,10 +86,32 @@ export default class Panel extends Component {
   fetchComments(kind, story) {
     getComments(kind, story)
       .then((data) => {
-        this.setState({ comments: data.comments });
+        var comments = data.comments,
```
Comment (leanne1) [inline src/components/panel/index.jsx]: `const` / `let` ? (My code editor moans about `var`)

File: src/components/panel/index.jsx [hunk 5]
```diff
@@ -81,10 +86,32 @@ export default class Panel extends Component {
...
+        }
+        this.setState({
+          comments: showingAllComments ? this.allComments : this.filteredComments,
+          showingAllComments: showingAllComments
+        });
         // add listener for channel comments events
         this.listenForCommentChanges();
-      }).catch((e) => {});
+      }).catch((e) => {
+        console.log('Error:', e);
```
Comment (leanne1) [inline src/components/panel/index.jsx]: Is this dev code?

**Comment by leanne1** [comment]
👍

File: src/components/comments/index.jsx (hunk 1 above)
Comment (jon-ec) [reply src/components/comments/index.jsx]: Yip, should have seen that!

**Pull request merged**